### DISCUSSION TITLE
pull-request buffer not ordered

### DISCUSSION THREAD

**Discussion by CDitzel** [magit/forge#468] [Support] [answered]
having pulled in all topics from my company gitlab servers repo, I see posts (gitlab threads) in the buffer after entering into one of the merge requests. However, comments seem to be ordered in strict chronological order in which they were originally posted which means that related posts on code sections turn up at completely different positions in the list. This makes it almost impossible to track a conversation. I expected comments which are grouped on gitlab which revolve around a certain section of code to be also grouped here. 

Also, I cannot figure out how/if forge shows code sections that are referred to in threads. 

Can someone help?

PS: for every PR that I enter, there is a commit section header but there is never anything outlined below it, maybe that is the problem?

**Comment by tarsius** [comment, marked as answer]
Forge doesn't actually support "review comments". A few third-party packages fill that cap:

- https://github.com/wandersoncferreira/code-review
- https://github.com/charignon/github-review (github only)
- https://github.com/blahgeek/emacs-pr-review (github only)

The reason the "review comments" show up among the "regular comments" when using Gitlab is that its API doesn't make it easy to tell those apart. At least it didn't last I checked.
